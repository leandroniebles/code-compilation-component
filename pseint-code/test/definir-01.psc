Proceso prueba
  definir a como entero;
  definir b como real;
  definir c como numero;
  definir d como numeros;
  definir e como número;
  definir f como números;
  definir g como numerico;
  definir h como numericos;
  definir i como numerica;
  definir j como numericas;
  definir k como numérico;
  definir l como numéricos;
  definir m como numérica;
  definir n como numéricas;
  a<-"lala";
  b<-"lala";
  c<-"lala";
  d<-"lala";
  e<-"lala";
  f<-"lala";
  g<-"lala";
  h<-"lala";
  i<-"lala";
  j<-"lala";
  k<-"lala";
  l<-"lala";
  m<-"lala";
  n<-"lala";
FinProceso