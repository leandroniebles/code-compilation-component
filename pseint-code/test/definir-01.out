=== Line 16: SynError 125
=== Line 17: SynError 125
=== Line 18: SynError 125
=== Line 19: SynError 125
=== Line 20: SynError 125
=== Line 21: SynError 125
=== Line 22: SynError 125
=== Line 23: SynError 125
=== Line 24: SynError 125
=== Line 25: SynError 125
=== Line 26: SynError 125
=== Line 27: SynError 125
=== Line 28: SynError 125
=== Line 29: SynError 125
